consumer-configuration/README.adoc

```diff
@@ -0,0 +1,347 @@
+= Configuring consumer groups in {product-long}
+ifdef::context[:parent-context: {context}]
+:context: configuring-consumer-groups
+
+[IMPORTANT]
+====
+{product-long} is currently available for Development Preview. Development Preview releases provide early access to a limited set of features that might not be fully tested and that might change in the final GA version. Users should not use Development Preview software in production or for business-critical workloads. Limited documentation is available for Development Preview releases and is typically focused on fundamental user goals.
+====
+
+// Purpose statement for the assembly
+[role="_abstract"]
+Consumer groups share large data streams generated by producers from a given topic in a Kafka instance.
+Grouping consumers scales consumption to keep up with the rate of data produced.
+Consumers within a group don’t read data from the same partition, but can receive data from one or more partitions.
+
+As a developer of applications and services, you can view all the consumer groups that have access to a particular Kafka instance in {product}.
+If required, use the {product} web console to reset the offsets of consumer groups or delete consumer groups.
+
+As an alternative to using the {product} web console, you can use the `rhoas` command-line interface (CLI) or the Kafka `kafka-consumer-groups.sh` tool to review and update consumer groups.
+
+//Additional line break to resolve mod docs generation error
+
+[id="con-adding-consumer-group-id_{context}"]
+== How to add a consumer to a consumer group
+
+A group ID is _required_ for a consumer to be able to join a consumer group.
+
+When you configure a consumer client application to access Kafka, you assign a group ID to associate it with a consumer group.
+All consumers with the same group ID belong to the consumer group with that ID.
+Consumer groups connected to a Kafka instance are presented in the {product} web console.
+
+The group ID is the _name_ of the consumer group.
+Add it as a property to your consumer configuration.
+The group ID can't be null or empty.
+
+.Example group ID property configuration for a consumer
+[source,properties]
+----
+group.id=my-group-id
+----
+
+.Property attributes
+[cols="25%,75%"]
+|===
+
+h|Type
+|String
+
+h|Default value
+|None
+
+h|Supported values
+|Letters (Aa-Zz), numbers (0-9), underscores ( _ ), or hyphens ( - )
+|===
+
+If you are using Kafka bin scripts to consume messages, you can use the `kafka-console-consumer.sh` tool to create a group ID:
+[source,subs="+quotes,+attributes"]
+----
+bin/kafka-console-consumer.sh --bootstrap-server __<bootstrap_server>__ --topic test-topic --group my-consumer-group
+----
+
+The `<bootstrap_server>` is the bootstrap server endpoint for your service account.
+You copy this information for the Kafka instance in the {product} web console by selecting the options menu (three vertical dots) and clicking *View connection information*.
+
+[NOTE]
+====
```

Comment:
I suggested a rewrite of this note in my previous comment. To what's below. Reason was to reorder some things that made it clearer for me, to avoid the empty "there is/are", etc. Take or leave whatever you like or don't.

>If you add more consumers than the number of partitions in a consumer group, the additional consumers don't help throughput but they function as standby consumers that can replace any consumers that stop functioning. The fewer consumers you use in order to meet throughput, the more resources you save.

Reply:
Tweaked a little. Throughput isn't an end in itself. High throughput is generally good. _The fewer consumers you use in order to meet your throughput target..._